src/txn/txn.c

```diff
@@ -2772,7 +2772,7 @@ __wt_txn_global_shutdown(WT_SESSION_IMPL *session, const char **cfg)
          * Perform rollback to stable to ensure that the stable version is written to disk on a
          * clean shutdown.
          */
-        if (use_timestamp) {
+        if (use_timestamp && !__wt_conn_is_disagg(session)) {
```

Comment:
Can we use a local variable like:

is_disagg = __wt_conn_is_disagg instead of calling it twice?

Reply:
Done in 3b0fb82.